astroquery/esa_hubble/core.py

```diff
@@ -0,0 +1,374 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+
+@author: Javier Duran
+@contact: javier.duran@sciops.esa.int
+
+European Space Astronomy Centre (ESAC)
+European Space Agency (ESA)
+
+Created on 13 Ago. 2018
+
+
+"""
+from astroquery.utils import commons
+from astropy import units
+from astropy.units import Quantity
+from astroquery.utils.tap.core import TapPlus
+from astroquery.utils.tap.model import modelutils
+from six.moves import urllib
+
+from . import conf
+
+__all__ = ['EsaHubble', 'EsaHubbleClass']
+
+
+class ESAHubbleHandler(object):
+
+    def __init__(self):
+        return
+
+    def get_file(self, url, filename=None, verbose=False):
+        if filename is None:
+            raise ValueError("filename must be specified")
+        urllib.request.urlretrieve(url, filename)
+        if "/" not in filename:
+            print("File " + str(filename) + " downloaded to current directory")
+        else:
+            print("File " + str(filename) + " downloaded")
+
+    def get_table(self, url, filename=None, output_format='votable',
+                  verbose=False):
+        if filename is None:
+            raise ValueError("filename must be specified")
+        urllib.request.urlretrieve(url, filename)
+        table = modelutils.read_results_table_from_file(filename,
+                                                        str(output_format))
+        return table
+
+
+Handler = ESAHubbleHandler()
+
+
+class ESAHubbleClass(object):
```

Comment:
This class should inherit from [`BaseQuery`](https://github.com/astropy/astroquery/blob/master/astroquery/query.py#L142)